manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Just For Today",
+    "description": "A Chrome Extension to help motivate people in recovery",
+    "version": "0.1.0",
+    "manifest_version": 3,
+    "chrome_url_overrides" : {
+        "newtab": "./main.html"
+    }
+}
```

Comment:
same empty line at end of file suggestion

Reply:
I am getting "end of file expected" when I insert '\\n'

Reply:
Hmm, ok, we probably have different linting rules set up. Don't worry about it for now.